src/client/providers/setInterpreterProvider.ts

```diff
@@ -56,7 +56,7 @@ function suggestionToQuickPickItem(suggestion: PythonPathSuggestion) : vscode.Qu
     return {
         label: suggestion.label,
         description: suggestion.type,
-        detail: suggestion.path
+        detail: utils.IS_WINDOWS ? suggestion.path.replace(/\\/g, "/") : suggestion.path
```

Comment:
@DonJayamanne added this as when testing on Windows I saw that you have to escape Windows backslashes when saving to `settings.json` (and I noticed you did the same in the [Wiki here](https://github.com/DonJayamanne/pythonVSCode/wiki/Python-Path-and-Version#python-version-used-for-intellisense-autocomplete-linting-formatting-etc))